### PR TITLE
Doc 1007

### PR DESCRIPTION
**Pull request by aburathoki**
## Link to JIRA ticket

https://hackney.atlassian.net/browse/DOC-1007

## Describe this PR

This includes all the implementation and tests related to the conversion of HEIC file type to a compatible format.
Please check the comments in the link ticket for more details

### *What is the problem we're trying to solve*

Allow officers to view HEIC files uploaded by residents in their browser. Please check the ticket for more details.
https://hackney.atlassian.net/browse/DOC-1007

### *What changes have we introduced*
New Component ImagePreview 
Add Image conversion logic in ImagePreview
Add Spinning wheel when image is loading
Add test to include HEIC file type
Update mock server config to accept HEIC file

#### _Checklist_

- [x] Added end-to-end (i.e. integration) tests where necessary e.g to test complete functionality of a newly added feature
- [ ] Added tests to cover all new production code
- [ ] Added comments to the README or updated relevant documentation _(add link to documentation)_, where necessary.
- [ ] Checked all code for possible refactoring
- [x] Code pipeline builds correctly

### *Follow-up actions after merging PR*

Check-in staging, demo to the team and get feedback.

### CODE REVIEW

File: src/components/ImagePreview.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        ) : (
+          <iframe src={downloadUrl} height="1000px" width="800px" />
+        )}
+
+        <figcaption className="lbh-body-s">
+          <strong>{document.extension?.toUpperCase()}</strong>{' '}
+          {humanFileSize(document.fileSizeInBytes)}{' '}
+          {/*<a href={`${downloadUrl}`} target="blank" className="lbh-link">*/}
+          {/*  Open in new tab*/}
+          {/*</a>*/}
```
Comment (bogdan-zaharia) [inline src/components/ImagePreview.tsx:63]: Should this be removed?

**Review by bogdan-zaharia — CHANGES_REQUESTED** [review]
Looks good, left a few comments

File: cypress/integration/view-and-manage-evidence.spec.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
-import dsFixture from '../../cypress/fixtures/document_submissions/get-png.json';
-
+import dsFixture from '../../cypress/fixtures/document_submissions/get-pdf.json';
```
[the remark this answers is not in the document]
Comment (aburathoki) [reply cypress/integration/view-and-manage-evidence.spec.ts:1]: reverted to `get-png.json` as it is not in the scope of change.

File: cypress/integration/view-and-manage-evidence.spec.ts [hunk 3]
```diff
@@ -208,4 +208,51 @@ describe('When a user inputs a validity date that is in the past', () => {
...
+    cy.login();
+
+    cy.intercept('PATCH', '/api/evidence/document_submissions', (req) => {
+      const body = {
+        ...dsFixtureHeic,
+        id: 456,
+        state: req.body.state,
+        staffSelectedDocumentTypeId: req.body.staffSelectedDocumentTypeId,
+      };
+      console.log(body);
```
[the remark this answers is not in the document]
Comment (aburathoki) [reply cypress/integration/view-and-manage-evidence.spec.ts:222]: removed console.log(body)

File: cypress/integration/view-and-manage-evidence.spec.ts [hunk 4]
```diff
@@ -208,4 +208,51 @@ describe('When a user inputs a validity date that is in the past', () => {
...
+    cy.get('.toReview a').eq(2).contains('Proof of ID').click();
+
+    cy.contains('h1', 'Namey McNameProof of ID');
+
+    cy.get('button').should('contain', 'Accept');
+    cy.get('button').should('contain', 'Request new file');
+    cy.get('h2').should('contain', 'Preview');
+    cy.get('svg[class="icon-loading"]').should('be.visible');
+    cy.contains('HEIC');
+    cy.contains('9.8 KB');
```
[the remark this answers is not in the document]
Comment (aburathoki) [reply cypress/integration/view-and-manage-evidence.spec.ts:245]: I have now specified element  in the test that captures the texts above.

File: src/components/ImagePreview.tsx (hunk 1 above)
Comment (aburathoki) [reply src/components/ImagePreview.tsx:63]: Agreed to remove this code as it is no longer in use.

**Review by aburathoki — COMMENTED** [review]
I have committed the suggested changes. Does this resolve the previous change request?

**Review by bogdan-zaharia — APPROVED** [review]

**Pull request merged**